pythermalcomfort/models/jos3.py

```
@@ -601,7 +601,7 @@ def _reset_setpt(self) -> JOS3Output:
             Parameters of the JOS-3 model.
         """
         # Set operative temperature under PMV=0 environment
-        # todo shall these be the rerefence values for naked?
+        # todo shall these be the reference values for naked?
```

Pylint (3.3.7)

[warning] todo shall these be the reference values for naked?

(W0511)